### PR TITLE
xz: update to 5.6.3

### PR DESCRIPTION
**Pull request by jen1-sys**
xz: update to 5.6.3.

Changes: https://github.com/tukaani-project/xz/releases/tag/v5.6.3

Maintainer: me / @krant 
Compile tested: Model	ASUS RT-AC88U
Architecture	ARMv7 Processor rev 0 (v7l)
BCM53xx / arm_cortex-a9.
Kernel Version	6.6.53
main branch
Run tested: Using GCC 14.2.0 and binutils 2.43.1

### CODE REVIEW

File: utils/xz/Makefile [hunk 1]
```diff
@@ -9,12 +9,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xz
-PKG_VERSION:=5.6.2
+PKG_VERSION:=5.6.3
 PKG_RELEASE:=2
```
Comment (BKPepe) [inline utils/xz/Makefile:13]:
```suggestion
PKG_RELEASE:=1
```

**Comment by jen1-sys** [comment]
@BKPepe  I deleted the file 010-libtool.patch, but didn't know how to do it here? can you help?

**Comment by BKPepe** [comment]
@jen1-sys You created more than 30 PRs for OpenWrt repositories despite none having merged so far. You will need to figure it out how you can remove files using Git.

**Comment by Neustradamus** [comment]
@jen1-sys: Have you seen the latest @BKPepe comment?

**Comment by BKPepe** [comment]
Closing due to inactivity.

**Pull request closed without merging**